src/Shared/Application/Command/PasswordResetTokenSeeder.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Shared\Application\Command;
+
+use App\User\Domain\Entity\PasswordResetToken;
+use App\User\Domain\Entity\UserInterface;
+use App\User\Domain\Repository\PasswordResetTokenRepositoryInterface;
+use DateTimeImmutable;
+use Doctrine\DBAL\Connection;
+
+final readonly class PasswordResetTokenSeeder
+{
+    public function __construct(
+        private Connection $connection,
+        private PasswordResetTokenRepositoryInterface $tokenRepository
+    ) {
+    }
+
+    /**
+     * @param array<int,string> $tokenValues
+     */
+    public function seedTokens(UserInterface $user, array $tokenValues): void
+    {
+        foreach ($tokenValues as $tokenValue) {
```

Comment:
Please review this method and the code
I don’t understand why do we need to remove the token and then add the same but with +1 hour value
It will be better to just update the old record to avoid unnecessary work